### DISCUSSION TITLE
Spotube doesn't installed on my macbook

### DISCUSSION THREAD

**Discussion by Patvc15** [KRTirtho/spotube#1042] [Q&A]
when i tried to install spotube i received this message:

“spotube” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

**Comment by Sirivasv** [comment]
Quoting this other [answer](https://github.com/KRTirtho/spotube/discussions/1097#discussioncomment-8203557) from [verhovsky](https://github.com/verhovsky)

> Go to System Settings > Privacy & Security > Security (scroll down) > Open Anyway > Open

<img width="715" alt="298364629-c2eeff81-fb96-47d4-ad16-c04c5945299f" src="https://github.com/KRTirtho/spotube/assets/6540632/630bba7a-2527-48e5-bb85-ebd0725bd03d">

**Reply by Patvc15** [reply]
Thank you for your help so far. After completing the previous step, I attempted to log in to the app using my Spotify credentials. However, I received the following error message:

_exception : failed to get access token : "error 401, message : unauthorized"_

<img width="812" alt="Screenshot 2024-02-01 at 13 22 37" src="https://github.com/KRTirtho/spotube/assets/139363514/0bca1b5c-c482-4846-9ef8-dd13b4e4d7a5">
